### ISSUE TITLE
Cycling through maps leads to errors and crashes

### ISSUE THREAD

**Issue by PentalginPlus** [dreamstalker/rehlds#852]
When a server cycles through some custom maps, you can encounter some errors and crashes.
Most notable error is a player stuck in a cycler_sprite that is set to an enormous size due to an error (it has no size by default). Sometimes models (cycler_sprite entities) become invisible, sometimes sprite entities like env_sprite are set to an invalid RenderFX.
Eventually, when you will switch to more maps it would lead to crashes like:
`L 25/07/2021 - 07:01:40: (map "de_KOHTPA") Cache_Free: not allocated`

**Steps to recreate:**

1. [Install maps and mapcycle.txt](https://www.mediafire.com/file/cici43f9o9h00wb/crash_maplist.zip/file) on a server
2. Set mp_timelimit to minute or less to cycle through maps faster, or change them manually. Uptime doesn't matter but number of loaded maps since the server startup does
3. Wait for a while or keep changing different maps manually until you will encounter the player stuck or another issue (pick different teams or walk for a while, you will notice being stuck or touching "invisible walls". Try it on **de_slc** for example)
4. Adding more custom maps to mapcycle will increase the chance of listed errors and crashes.

**version**
```
Protocol version 48
Exe version 1.1.2.7/Stdio (cstrike)
ReHLDS version: 3.10.0.760-dev
Build date: 13:06:48 Jun 23 2021 (2629)
Build from: https://github.com/dreamstalker/rehlds/commit/9508c83
```
**game version**
```
ReGameDLL version: 5.20.0.516-dev
Build date: 21:01:56 Jun 14 2021
Build from: https://github.com/s1lentq/ReGameDLL_CS/commit/2c52c4f
```
**mapcycle.txt**
```
as_resident_evil
cs_KVARTAL2
cs_agency_csgo
cs_cruise
cs_evilcrazy
cs_greatassault_b3
cs_paranoia
cs_xmas_house
csde_cold_college
csde_grozniy
de_02shkola
de_KOHTPA
de_gulag_final
de_gyumri
de_pycb
de_santorini
de_slc
fun_matrix_reloaded
```
Tested **without metamod**.

**Comment by Lopol2010** [comment]
I think i had the same issue. 
1. Once players completely stuck (air on map become solid). 
2. And once  invisible walls appeared on another map, but you can walk. Map mrcn_bom.
3. And once some players reported they completly stuck on another map but i didnt saw this one.

All these cases was on very start of the maps
And if i remember correctly the only fix for this is hard reset; kill hlds process and start again. Disabling all plugins and command server to restart didnt help.

**Comment by RauliTop** [comment]
@PentalginPlus @Lopol2010
Are you tested on HLDS?

Try downgrading rehlds version too. To find what version has this error.

**Comment by Lopol2010** [comment]
@RauliTop in my case it was like 3 maps over a year, not critical.

Also i think would be good if @PentalginPlus can clarify, do these stucks and invisible walls preserve even after you restart or change map to the same map?

**Comment by PentalginPlus** [comment]
@Lopol2010 
> do these stucks and invisible walls preserve even after you restart or change map to the same map?

Yes, it stays all the same even if you change the map to the same. The only way to get rid of it is to restart the rehlds process
@RauliTop 
>Are you tested on HLDS?
> Try downgrading rehlds version too. To find what version has this error.

I encountered this kind of issue for the first time about two years ago, so it's not a something that went wrong after an update. Didn't test it on HLDS
